src/App.tsx

```diff
@@ -1,16 +1,41 @@
 import React from 'react';
 import './App.scss';
 
-class App extends React.Component {
-  addOne = () => {};
+type State = {
+  count: number,
+};
 
-  add100 = () => {};
+class App extends React.Component<{}, State> {
+  state:State = {
+    count: 0,
+  };
 
-  increase = () => {};
+  addToCounter = (num = 1):void => {
+    this.setState(({ count }) => (
+      {
+        count: count + num,
+      }
+    ));
```

Comment:
не переноси так скобки, исправь ниже
```suggestion
    this.setState(({ count }) => ({
      count: count + num,
    }));
```